FirebaseAuthUI/README.md

```diff
@@ -79,23 +81,28 @@ This instance can then be configured with the providers you wish to support:
 // swift
 import FirebaseGoogleAuthUI
 import FirebaseFacebookAuthUI
-
-let googleAuthUI = FIRGoogleAuthUI(clientID: kGoogleClientID)
-let facebookAuthUI = FIRFacebookAuthUI(appID: kFacebookAppID)
-
-authUI?.providers = [googleAuthUI, facebookAuthUI]
+import FirebaseTwitterAuthUI
+
+let providers: [FIRAuthProviderUI] = [
+  FIRGoogleAuthUI(),
+  FIRFacebookAuthUI(),
+  FIRTwitterAuthUI(),
+]
+self.authUI?.providers = providers
 ```
 
 ```objective-c
 // objc
 @import FirebaseGoogleAuthUI
 @import FirebaseFacebookAuthUI
+@import FIRTwitterAuthUI
 ...
-FIRGoogleAuthUI *googleAuthUI =
-    [[FIRGoogleAuthUI alloc] initWithClientID:kGoogleClientID];
-FIRFacebookAuthUI *facebookAuthUI =
-    [[FIRFacebookAuthUI alloc] initWithAppID:kFacebookAppID];
-authUI.signInProviders = @[ googleAuthUI, facebookAuthUI];
+NSArray<id<FIRAuthProviderUI>> *providers = [NSArray arrayWithObjects:
+                                             [[FIRGoogleAuthUI alloc] init],
+                                             [[FIRFacebookAuthUI alloc] init],
+                                             [[FIRTwitterAuthUI alloc] init],
+                                             nil];
```

Comment:
Instead of using `arrayWithObjects:` this could be written as

```objc
@[
  [[FIRGoogleAuthUI alloc] init],
  [[FIRFacebookAuthUI alloc] init],
  [[FIRTwitterAuthUI alloc] init],
];
```